modules/gateway/nodes.go

```diff
@@ -18,6 +18,18 @@ var (
 	errPeerGenesisID = errors.New("peer has different genesis ID")
 )
 
+// A node represents a potential peer on the Sia network. The Prioritize field
+// indicates whether the node should be prioritized when forming new
+// connections.
+type node struct {
+	NetAddress modules.NetAddress `json:"netaddress"`
+	Prioritize bool               `json:"prioritize"`
```

Comment:
I considered `Trusted` too but rejected it for the same reason -- too strong a word.
I'll go with `WasOutboundPeer` for now.